scripts/ff_ci_pr_build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
So I went ahead and renamed this script to spell out `ff` as `fast_finish` so it shows up clearly in the UI and is hopefully self-evident combined with the docs in the actual script.